examples/ai/edge-functions/supabase/functions/generate-embedding/index.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="https://esm.sh/v135/@supabase/functions-js@2.3.0/src/edge-runtime.d.ts" />
```

Comment:
can we change the URL here to `https://esm.sh/@supabase/functions-js/src/edge-runtime.d.ts` it will be redirected to above. That way we don't need to update all references on a new release.

Reply:
I generally like to specify the exact versions as that means the code won't break with future changes. But I've updated it to remove.